test/unittests/mongoBackend/mongoOntimeintervalOperations_test.cpp

```diff
@@ -522,8 +526,10 @@ TEST(mongoOntimeintervalOperations, mongoGetContextElementResponses_dbfail)
               "attrNames: { $in: [ \"A1\", \"A2\", \"A3\", \"A4\" ] } }, orderby: { creDate: 1 } } - "
               "exception: boom!!)", err);
 
-    /* Release mock */
-    setMongoConnectionForUnitTest(NULL);
+    /* Restore real DB connection */
+    setMongoConnectionForUnitTest(connectionDb);
```

Comment:
All these *setMongoConnectionForUnitTest(connectionDb);* are because utInit/utExit aren't used in these functions, right?